src/simple_pathfinding.cpp

```diff
@@ -192,10 +193,10 @@ struct node_address {
 
 struct node_address_hasher {
     std::size_t operator()( const node_address &addr ) const {
-        std::size_t val = addr.x;
+        std::int64_t val = addr.x;
```

Comment:
Better use unsigned types in the scenario of hashing and bitwise operations.

Reply:
Done.